src/sentry/tasks/deletion.py

```diff
@@ -56,12 +73,10 @@ def run_deletion(deletion_id):
     if deletion.aborted:
         raise DeleteAborted
 
-    if not deletion.in_progress:
```

Comment:
This branch was unreachable before as `schedule_deletions` sets `in_progress = True`